atc/api/resources_test.go

```diff
@@ -1128,17 +1096,15 @@ var _ = Describe("Resources API", func() {
 						})
 
 						It("tries to scan with the latest version", func() {
-							Expect(fakeScanner.ScanFromVersionCallCount()).To(Equal(1))
+							Eventually(fakeScanner.ScanFromVersionCallCount()).Should(Equal(1))
```

Comment:
This `Eventually` will block until it either receives the expect result or times out. You shouldn't need the other `Eventually` blocks below since the `fakeScanner` will already have been invoked.

Reply:
You also need to remove the `()` from the call otherwise it'll just be polling the result of the first call:

```							Eventually(fakeScanner.ScanFromVersionCallCount).Should(Equal(1))
```

Reply:
Makes sense. Thanks!